src/ui/reducers/app.ts

```diff
@@ -298,6 +300,17 @@ export const getLoading = (state: UIState) => state.app.loading;
 export const getDisplayedLoadingProgress = (state: UIState) => state.app.displayedLoadingProgress;
 export const getLoadingFinished = (state: UIState) => state.app.loadingFinished;
 export const getLoadedRegions = (state: UIState) => state.app.loadedRegions;
+export const getNonLoadingTimeRanges = (state: UIState) => {
+  const loadingRegions = getLoadedRegions(state)?.loading || [];
+  const endTime = getZoomRegion(state).endTime;
+
+  return [
+    { start: 0, end: 5000 },
+    { start: 10000, end: 24555 },
+  ];
```

Comment:
Hack I added for now just to let me test how the UI feels. Technically I could have unloaded regions from the console but it's late and that takes more thinking time :P

Reply:
Will remove this prior to asking for a review